.github/workflows/release.yml

```diff
@@ -56,28 +93,57 @@ jobs:
             path: "Refresher/bin/Release/net8.0-windows/win-x64/publish/"
             if-no-files-found: error
             retention-days: 1
+
+      - name: Upload macOS x64 build
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v3.1.1
+        with:
+            name: "Refresher_for_macOS_x64"
+            path: "Refresher-x64.tar"
+            if-no-files-found: error
+            retention-days: 1
+  
+      - name: Upload macOS ARM64 build
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v3.1.1
+        with:
+            name: "Refresher_for_macOS_ARM64"
+            path: "Refresher-arm64.tar"
+            if-no-files-found: error
+            retention-days: 1
   release:
     name: Release Built Artifacts
     needs: build
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
+        id: src
+
       - name: Download artifacts
         id: download
         uses: actions/download-artifact@v3.0.2
 
       - name: Create zip files
         run: |
             cd ${{steps.download.outputs.download-path}}/Refresher_for_Windows_x64
-            zip -r "${{steps.download.outputs.download-path}}/Refresh_for_Windows_x64.zip" .
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_Windows_x64.zip" .
             
             cd ${{steps.download.outputs.download-path}}/Refresher_for_Linux_x64
-            zip -r "${{steps.download.outputs.download-path}}/Refresh_for_Linux_x64.zip" .
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_Linux_x64.zip" .
+
+            cd ${{steps.download.outputs.download-path}}/Refresher_for_macOS_x64
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_macOS_x64.zip" .
+
+            cd ${{steps.download.outputs.download-path}}/Refresher_for_macOS_ARM64
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_macOS_ARM64.zip" .
         
-      - uses: marvinpinto/action-automatic-releases@latest
+      - uses: ncipollo/release-action@v1.14.0
         name: "Create release"
         with:
-          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          token: "${{ secrets.GITHUB_TOKEN }}"
           prerelease: false
           draft: true
-          files: |
+          generateReleaseNotes: true
+          bodyFile: "RELEASE_NOTES.md"
+          artifacts: |
             *.zip
```

Comment:
Wouldn't this exclude the .tar files?

Reply:
excluding the tarballs is intended behaviour, since they are zipped into `.tar.zip` in previous steps (to preserve the permission bits)